debian/changelog

```diff
@@ -0,0 +1,25 @@
+ckb-next (0.4.0-snapshot) zesty; urgency=low
```

Comment:
Do we really need a second changelog?
Isn't it possible to generate this one from the main one? Or at least make a dummy one that's auto generated for packaging.

I really do not want to have to maintain a second changelog.